### PR TITLE
adding a yoga app with music for pranayam yoga

### PR DESCRIPTION
**Pull request by sidmohanty11**
## Related Issue
Yoga App

Closes: #397

### Describe the changes you've made
Made this to further simplify yoga and not to go and search around books or complex apps for meditation purposes.

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] New and existing unit tests pass locally with my changes.

### Screenshots
yogass

### Event
gssoc'21

### CODE REVIEW

**Review by urvashi-code1255 — COMMENTED** [review]
@sidmohanty11 Its looking good, just change the image to GIF

**Comment by sidmohanty11** [comment]
can you check ma'am? I also added another project flip cards. by committing everything. Can it be done like this or should I delete them and make a new request?

**Review by urvashi-code1255 — COMMENTED** [review]
Yes @sidmohanty11 You can do it like this. Everything is fine. 

Please resolve the merge conflicts so that I can merge this. 

Please tag me when you are done with the changes

**Comment by sidmohanty11** [comment]
Need some help with this

**Comment by sidmohanty11** [comment]
ma'am its done now sorry for delay.

**Comment by sidmohanty11** [comment]
ma'am ive been merging conflicts three times from yesterday...Can you please review this?

**Comment by 991rajat** [comment]
@sidmohanty11   Good Work
Few suggestions tell me if you like it or anything else you would like to suggest
Make the song play indefinitely. Once user plays it doesn't stop until user stop it again. Like ZEN Mode

Squash the commits please.
Let me know if you need any help.

**Comment by sidmohanty11** [comment]
@991rajat Sir can you review my code? I added that feature

File: public/yoga.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <p id="text"></p>
+
+      <div class="pointer-container">
+        <span class="pointer"></span>
+      </div>
+
+      <div class="gradient-circle"></div>
+    </div>
+    <!-- <audio src="../assets/sounds/yoga-beat.mp3" controls></audio> -->
+    <embed name="yoga-beat" src="../assets/sounds/yoga-beat.mp3" width="150" height="80" loop="true" autostart="false">
```
Comment (991rajat) [inline public/yoga.html:25]: Don't use embed,
Use audio tag as it's audio it's playing 
image

**Review by 991rajat — CHANGES_REQUESTED** [review]
Make it a little more responsive 
And Squash the commits Please 
Let me know if you need any help

**Comment by sidmohanty11** [comment]
Thank you so much sir for your feedback @991rajat , I made it responsive. Can you please review my code now?

File: public/yoga.html (hunk 1 above)
Comment (sidmohanty11) [reply public/yoga.html]: Thank you sir for your valuable feedback, I have changed the code accordingly.

**Comment by sidmohanty11** [comment]
@urvashi-code1255  ma'am can you please review my code??

**Review by Vishal-raj-1 — COMMENTED** [review]
Are you making three project in this PR ?

**Comment by Vishal-raj-1** [comment]
Okay, Everything Looks perfect to me !! @sidmohanty11 Learn How to squash commits, from next time I will merge your PR iff commits are squashed. 

Thanks for your contribution, Hoping you have stared this project.

**Pull request merged**